src/components/select/simple-select/simple-select.pw.tsx

```diff
@@ -996,6 +997,93 @@ test.describe("SimpleSelect component", () => {
     });
   });
 
+  ([
+    ["top", "100px 300px", 150, 2],
+    ["bottom", "0px 300px", 150, 68],
+    ["top-end", "100px 300px", 150, 2],
+    ["bottom-end", "0px 300px", 150, 68],
+    ["top-start", "100px 0px", 0, 2],
+    ["bottom-start", "100px 0px", 0, 168],
+  ] as [
+    NonNullable<SimpleSelectProps["listPlacement"]>,
+    string,
+    number,
+    number
+  ][]).forEach(([position, margin, left, top]) => {
+    test(`should render list wider than the input when listWidth is 350 and positioned based on listPlacement of ${position}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <ListWidth margin={margin} listPlacement={position} listWidth={350} />
+      );
+
+      const positionValue =
+        !position.includes("end") && !position.includes("start")
+          ? `${position}-end`
+          : position;
```

Comment:
**suggestion(non-blocking)**: what do you think about defining this value as a parameter for the tests? This would avoid the need for conditional logic within the test:

```tsx
([
  ["top", "top-end", "100px 300px", 150, 2],
  ["bottom", "bottom-end", "0px 300px", 150, 68],
] as const).forEach(([position, expectedPosition, margin, left, top] => {
   // test code...
});
```

Reply:
I think I'll leave this for now as there is other examples of the same approach in this file so I'd prefer it be tidied up by the code owners when they come to review the suite as a whole etc